pkg/sql/parser/builtins.go

```diff
@@ -580,16 +642,35 @@ var Builtins = map[string][]Builtin{
 				sqlFlags := string(*args[3].(*DString))
 				return regexpReplace(ctx, s, pattern, to, sqlFlags)
 			},
+			Info: "Replaces matches for the Regular Expression `regex` in `input` with the " +
+				"Regular Expression `replace` using `flags`.<br/><br/>CockroachDB supports the " +
+				"following flags:<br/><br/>&#8226; **c**: Case-sensitive matching<br/><br/>" +
+				"&#8226; **g**: Global matching (match each substring instead of only the " +
+				"first).<br/><br/>&#8226; **i**: Case-insensitive matching<br/><br/>&#8226; " +
+				"**m** or **n**: Newline-sensitive matching: `.` and negated brackets (`[^...]`)" +
+				" brackets do not match newline characters (preventing matches from crossing " +
+				"newlines unless explicitly defined to); `^` and `$` match the space before and" +
+				"after newline characters respectively (so characters between newline characters" +
+				"are treated as if they're on a separate line).<br/><br/>&#8226; **p**: Partial" +
```

Comment:
Need space after "Partial". Otherwise, this gets rendered as "Partialnewline-sensitive".